src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1366,14 +1384,25 @@ subroutine von_Mises_calving(domain, err)
              if ( .not. li_mask_is_floating_ice(cellMask(iCell)) ) then
                 calvingVelocity(iCell) = sqrt(xvelmean(iCell)**2.0_RKIND + yvelmean(iCell)**2.0_RKIND) * &
                                          vonMisesStress(iCell) / config_grounded_von_Mises_threshold_stress
-             elseif ( li_mask_is_floating_ice(cellMask(iCell)) ) then
-                calvingVelocity(iCell) = sqrt(xvelmean(iCell)**2.0_RKIND + yvelmean(iCell)**2.0_RKIND) * &
+             elseif ( li_mask_is_floating_ice(cellMask(iCell)) .and. config_floating_von_Mises_threshold_stress .ne. 0.0_RKIND) then
```

Comment:
Can you add a comment for why the mask might indicate floating but the vM stress is 0?  Does the vM stress==0 case need to be handled differently?

Reply:
I'm not entirely sure what you mean by this. This is referring to the VM stress threshold for floating ice, not the calculated VM stress. If config_floating_von_Mises_threshold_stress is set to 0.0, then we simply remove floating ice. Otherwise, we calculate a calving velocity. 
Or are you asking why there is ever floating ice when we use floatkill?

Reply:
Oh my bad, I was reading it as the vM stress itself, but it's clearly the config threshold.  In that case just add a short comment that floating vM threshold=0 is handled below.

Also, at the top of this routine, add a check and error if config_grounded_von_Mises_threshold_stress==0.0.

Reply:
Comment and check/error added.